<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>249</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>kind</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>ID</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>address</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLineEdit" name="kindEdit"/>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="IDEdit"/>
        </item>
        <item row="0" column="2">
         <widget class="QLineEdit" name="addressEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="1" colspan="4">
       <widget class="QDateTimeEdit" name="dateTimeEdit"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>随机数上限</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>时间上限</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>时间下限</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>随机数下限</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>插入频率</string>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>条/小时</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="4">
       <widget class="QDateTimeEdit" name="dateTimeEdit_2"/>
      </item>
      <item row="3" column="2" colspan="3">
       <widget class="QLineEdit" name="downEdit"/>
      </item>
      <item row="2" column="2" colspan="3">
       <widget class="QLineEdit" name="UPlineEdit"/>
      </item>
      <item row="4" column="2" colspan="2">
       <widget class="QLineEdit" name="HZEdit"/>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>开始插入</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>225</width>
         <height>115</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>249</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
